packages/resolve-redux/src/create_reducer.js

```diff
@@ -2,13 +2,13 @@ import { MERGE } from './actions';
 
 export default function createReducer({ name, eventHandlers }, extendReducer) {
     const handlers = {
-        ...eventHandlers,
         [MERGE]: (state, action) => {
             if (action.projectionName === name) {
                 return state.merge(action.state);
```

Comment:
In any case using merge function is an implicit dependency. There is need to include seamless-immutable as a dependency or use object.assign instead